web/cobrands/sass/_base.scss

```diff
@@ -1496,6 +1497,48 @@ html.js #map .noscript {
       background-image:url($image-sprite);
       background-position: flip(right, -341px) -3936px;
     }
+    &.expand {
+      padding: 0.6em 1em 0.5em 1em;
+      &:after {
```

Comment:
Not saying this is wrong, but why is this done differently to the other sub map link icons, with an `:after`, and not just an immediate background-image?
Also, note how the other icons do their positioning - you need to account for the LTR/RTL possibility (where you'd want the icon on the right in RTL).